src/index.ts

```diff
@@ -1,16 +1,24 @@
 import * as dataTypes from './dataTypes';
-import { Map, TileLayer, GeoJSON } from 'leaflet';
+// TODO karma complains if import from 'geoJson'. Figure out more elegant solution.
```

Comment:
Hey @justindra, have you ever encountered this problem with  Karma not resolving `@type`  imports? 

Also, if you have the interest, this code could probably benefit from another pair of eyes. Unfortunately I mixed the PR up with some housekeeping surrounding the imports, so that distracts a little. This project is basically to abstract some of the imperative code required for adding layers in Leaflet. Im trying to make a plugin  to make layer adding easy peasy through configuration. Hopefully it can be used with another open source project to assist with drying and refactoring.

Reply:
Hm... I haven't used Karma with TS before. So never run into this issue. Is this the same reason why you decided to change from a importing a specific type from `geoJson` and `leafletTypes` to importing all using `*`? I would say its something to do with if Karma is using a different TS configuration to your own tsconfig?

I did find this on SO, might be helpful?
https://stackoverflow.com/questions/43161453/karma-typescript-cannot-find-module

Reply:
I started importing with `* as ___` because I wanted to be able to keep track of which type came from where. For example, Leaflet has a type of `GeoJson` which, if I remember correctly is a type of Leaflet layer, but then I also want to pull things from a `geojson` library that have names that sound very Leaflet-y.

Do you see any folly in this approach?

As for `@Type` imports, I think I need to pull myself out of the rabbit hole of fixing it, and just live with importing from file paths. Now its not just Karma complaing about it, it is the typescript compiler too. It tells me `Cannot import type declaration files. Consider importing 'geojson' instead of '@types/geojson'.` which I dont like. I dont want the geojson library, I just want the types at the moment and I think one should be able to import `@types` like they import anything else. Oh well.

Reply:
After looking at the imports in your project, I see the folly with my `import * as` approach. It means I am bundling up a bunch of unnecessary dependencies! I will probably explore something like `import {X, Y as leafletY, Z}` as an approach.

Reply:
Hm... Yea, that is the main reason why I would import each individual item. But at the same time, your bundler needs to be configured to do that properly too. I've had issues with it before where it was still just bundling everything in anyways.